ros2_control_demo_robot/description/diffbot_system.urdf.xacro

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0"?>
+<!-- Revolute-Revolute Manipulator -->
+<!--
+Copied and modified from ROS1 example:
+https://github.com/ros-simulation/gazebo_ros_demos/blob/kinetic-devel/diffbot_description/urdf/diffbot.xacro
+-->
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro" name="2dof_robot">
+
+  <!-- Import diffbot macro -->
+  <xacro:include filename="diffbot/diffbot.urdf.xacro" />
+  <!-- <xacro:include filename="$(find ros2_control_demo_robot)/description/diffbot/diffbot.urdf.xacro" /> -->
+
+  <!-- Import all Gazebo-customization elements, including Gazebo colors -->
+  <xacro:include filename="diffbot/diffbot.gazebo.xacro" />
+  <!-- <xacro:include filename="$(find ros2_control_demo_robot)/description/diffbot/diffbot.gazebo.xacro" /> -->
```

Comment:
you should be able to use this commands. They are prefered.

Reply:
Ok. I was following the rrbot example but I understand they are prefered.